Add realm-cli instructions (#37)
realm/realm-dart-samples: flutter_flexible_sync/README.md

## Atlas App Services Configuration Steps

### Using existing demo App Service

This sample is using an already prepared Atlas App Service with AppID `flutter_flexible_sync-rmjux`.
The app_id is configured in "\assets\atlas_app\realm_config.json"

### Creating a new App Service

If you want to create an Atlas App Service and have an access to the cloud App, follow the instruction below.

1. Create an account on [cloud.mongodb.com](https://cloud.mongodb.com). Follow the instructions: [Register a new Atlas Account](https://www.mongodb.com/docs/atlas/tutorial/create-atlas-account/#register-a-new-service-account).

#### Using Realm CLI
1. Create an App using [realm-cli](https://www.mongodb.com/docs/atlas/app-services/cli/#mongodb-binary-bin.realm-cli).

> 1. Open command line terminal and go to the root folder of this Flutter app.

Reviewer: ```suggestion
1. Open a terminal in the root folder of the sample.
```

1. Install `realm-cli` following the [instructions](https://www.mongodb.com/docs/atlas/app-services/cli/#mongodb-binary-bin.realm-cli).

    `npm install -g mongodb-realm-cli`

1. Login to the realm-cli:

    `realm-cli login --api-key="<my api key>" --private-api-key="<my private api key>"`

1. Go into folder '\assets\atlas_app':

    `cd assets/atlas_app`

1. Deploy the app to Atlas App Services:

> * IMPORTANT: Before to push the app make sure the cluster name is the same like the cluster in your account. Go to "\assets\atlas_app\data_sources\mongodb-atlas\config.json" and set the json field `clusterName`.

Reviewer: ```suggestion
* IMPORTANT: Before pushing the app make sure the cluster name is the same like the cluster in your account. Go to "\assets\atlas_app\data_sources\mongodb-atlas\config.json" and set the json field `clusterName`.
```
